app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,93 @@
+require 'pry'
+class TweetsController < ApplicationController
+    
+   
+    get '/tweets' do
+        if !logged_in?
+          redirect '/login'
+      else 
+        @tweets = Tweet.all 
+        erb :'tweets/index'
+        end
+    end 
+    
+    
+    get '/tweets/new' do
+        if logged_in?
+        erb :'tweets/new' #show new tweets view
+    else 
+        redirect '/login'
+        end
+    end 
+    
+    
+    post '/tweets' do
+     if logged_in?
+        if params[:tweet][:content] != ""
+            @tweet = Tweet.new(params[:tweet]) #create new tweet
+            @tweet.user_id  = current_user.id #set session to the users id
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```